packages/contracts-bedrock/contracts/deployment/MigrationSystemDictator.sol

```diff
@@ -143,14 +143,19 @@ contract MigrationSystemDictator is BaseSystemDictator {
      * @notice Upgrades and initializes proxy contracts.
      */
     function step5() external onlyOwner step(5) {
+        // Dynamic config must be set before we can initialize the L2OutputOracle.
+        require(dynamicConfigSet, "FreshSystemDictator: dynamic config not yet initialized");
```

Comment:
This contract is called `MigrationSystemDictator`

Reply:
Oops, copy/paste error